### DISCUSSION TITLE
Events API post request returns "ok"

### DISCUSSION THREAD

**Discussion by MarkusKeel** [plausible/analytics#3303] [Self-Hosted Support]
I am trying to use the events API to get our tracking data out of Plausible to use for analysis. I think I am able to connect to API and request the data but it is not returned. I have been looking at the status code and I get 202 as a status code which means that the request was accepted but doesn't mean that anything will be done with it necessarily(https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202). Any suggestions as to how I can get around this?

```
import requests
import json

bearer_token = "TOKEN"

url = "URL"

payload = json.dumps({
  "name": "pageview",
  "url": "URL",
  "domain": "DOMAIN",
  "props": {
     ...
  }
})

headers = {
  'Content-Type': 'application/json',
  'Authorization': f'Bearer {bearer_token}'
}

response = requests.post(url, headers=headers, data=payload)

print(response.status_code)
print(response.text)

```

**Comment by ruslandoga** [comment]
👋 @MarkusKeel 

What URL are you using for the request?
